src/SDKs/GuestConfiguration/Management.GuestConfiguration/Microsoft.Azure.Management.GuestConfiguration.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```

Comment:
I doubt you are going to need this file, all these properties need to be set in the csproj instead

Reply:
removed file